ValueMod Decimal fix added

AwesomeTornado opened the pull request:
ValueMod Decimal fix added.

Prevents a crash when the B input of the ValueMod\<Decimal\> node is set to zero or disconnected, https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/2746

Banane9 commented on CommunityBugFixCollection/ValueModDecimal.cs:
```diff
@@ -0,0 +1,28 @@
+
+        private static bool Prefix(Decimal a, Decimal b)
+        {
+            if (b == 0)
```
You need to add `Enabled && ` here for the toggling to work

Banane9 replied:
Really, you can just replace the whole Prefix implementation with `=> !Enabled || b != 0;`

Banane9 commented on README.md:
```diff
@@ -51,6 +51,7 @@ just disable them in the settings in the meantime.
 * Direct cursor size becoming very large when snapped to an object much closer than the true cursor (https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/3654)
 * Grid World floor shaking on AMD and Intel GPUs (https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/3829)
 * DuplicateSlot ProtoFlux node crashes game when if OverrideParent is identical to Template (https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/3950)
+* ValueMod<Decimal> node crashes the game when B input is set to zero or disconnected. (https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/2746)
```
I've kept them ordered by issue number, please move it to the right spot

Banane9 commented:
Also, please don't forget to write a description (and optionally name) in Locale/en.json - if you know any other languages, adding that would be great too

AwesomeTornado commented:
I have inlined the Prefix and added the enabled check, updated the readme so it should be sorted properly, added my mod description to the locale, and added myself as an author to the locale.

Should be ready now.
